packages/shopify-app-session-storage-postgresql/src/__tests__/postgresql.test.ts

```diff
@@ -42,13 +60,18 @@ describe('PostgreSQLSessionStorage', () => {
       {interval: 500, timeout: 20000},
     );
     storage = new PostgreSQLSessionStorage(dbURL);
+    storage2 = new PostgreSQLSessionStorage(dbURL2);
+
     await storage.ready;
   });
 
   afterAll(async () => {
     await storage.disconnect();
+    await storage2.disconnect();
+
     await exec(`podman rm -f ${containerId}`);
   });
 
   batteryOfTests(async () => storage);
+  batteryOfTests(async () => storage2);
```

Comment:
```suggestion
  const tests = [
    {dbName: 'shopitest', sessionStorage: async () => storage},
    {dbName: 'shopitest2', sessionStorage: async () => storage2},
  ];

  for (const {dbName, sessionStorage} of tests) {
    describe(`with ${dbName}`, () => {
      batteryOfTests(sessionStorage);
    });
  }
```
It's a bit of a nit bit if you structure the call to the tests like this, you'll end up with a more descriptive test output, like
![Screenshot 2022-12-01 at 2 23 14 PM](https://user-images.githubusercontent.com/56687600/205142071-bd701075-89c6-421a-bf37-c92ab8d20150.png)